### PR TITLE
Cleanup scrapers WIP

### PR DESCRIPTION
**Pull request by dajinchu**
Add types and get rid of the termdump, going straight to the database

### CODE REVIEW

File: backend/scrapers/types.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+// A block of meetings, ex: "Tuesdays+Fridays, 9:50-11:30am"
+export interface Meeting {
+  startDate: number,
+  endDate: number,
+  where: string,
+  type: string,
+  times: {
+    '1'?: MeetingTime,
```
Comment (ryanhugh) [inline backend/scrapers/types.ts]: There's actually a few classes that meet on the weekends. For those 
0 = sunday
6 = sat
*I think*

Comment (dajinchu) [reply backend/scrapers/types.ts]: :|

Comment (dajinchu) [reply backend/scrapers/types.ts]: good catch

**Review by gamburgm — APPROVED** [review]
:+1:

**Pull request merged**